internal/characters/traveler/common/hydro/travelerhydro.go

```diff
@@ -36,15 +36,25 @@ func NewChar(gender int) core.NewCharacterFunc {
 }
 
 func (c *char) Init() error {
-	c.droplets = make([]*common.SourcewaterDroplet, 0)
-
 	return nil
 }
 
+func (c *char) getSourcewaterDroplets() []combat.Gadget {
```

Comment:
the function returns the number of droplets created by traveler. add the condition `g.GadgetTyp() == combat.GadgetTypSourcewaterDropletHydroTrav`.

Reply:
I modified the documentation to instead say "number of sourcewater droplets on the field"